interface/src/Menu.h

```diff
@@ -35,6 +35,7 @@ namespace MenuOption {
     const QString AnimDebugDrawDefaultPose = "Debug Draw Default Pose";
     const QString AnimDebugDrawPosition= "Debug Draw Position";
     const QString AssetMigration = "ATP Asset Migration";
+    const QString AssetServer = "Asset Server";
```

Comment:
Let's change the name of this as well. Not sure what we should call it.. "Asset Browser" ?